package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "point-tokenization-vault-scripts",
+  "module": "index.ts",
+  "type": "module",
+  "scripts": {
+    "generate-merkle-tree": "bun run js-scripts/generateRedemptionRights/index.ts"
```

Comment:
bun 👀

Reply:
lol yes – i actually met the founder of bun at a wedding and it got me to try it out. It's p nice and all-in-one for typescript! Plus I like `bun build` for self-contained scripts